Framework/Kernel/src/ArrayProperty.cpp

```diff
@@ -1,103 +1,9 @@
 #include "MantidKernel/ArrayProperty.h"
 
-// PropertyWithValue implementation
-#include "MantidKernel/PropertyWithValue.tcc"
-
+// ArrayProperty Definition
+#include "MantidKernel/ArrayProperty.tcc"
```

Comment:
I get a whole bunch of compiler warnings otherwise about not being able to find the definition of `ArrayProperty<int>`